src/Simplex/Chat.hs

```diff
@@ -1312,7 +1312,45 @@ processChatCommand = \case
     case conn'_ of
       Just conn' -> pure $ CRConnectionIncognitoUpdated user conn'
       Nothing -> throwChatError CEConnectionIncognitoChangeProhibited
+  APICheckConnectionRequestURI userId (ACR SCMInvitation cReq) -> do
+    -- TODO [repeat connect]
+    -- - search connections (getConnectionEntity?) where conn_req_inv = cReq (and user_id? if yes add to idx_connections_conn_req_inv)
+    --   - if connection not found, return CRUInvitationLinkOkToConnect
+    --   - if connection entity is not RcvDirectMsgConnection, throw chat error prohibited (?)
```

Comment:
better not throw errors but return sum type - easier to handle.

Reply:
practically it shouldn't happen - it can happen if some other type of connection entity had conn_req_inv assigned (e.g. I think we have it temporarily for some group connections) and user tried to manually use that link - it's possible but they'd have to retrieve the link from database. I don't think it should be accounted for in ConnectionPlan